_tasks/build.ts

```diff
@@ -11,9 +11,9 @@ const wasmPaths = [
 ]
 
 let success = true
-await Promise.all(wasmPaths.map((path) =>
-  build(path).catch((error) => {
-    console.error(`${path}:`, error)
+await Promise.all(wasmPaths.map((wasmPath) =>
+  build(wasmPath).catch((error) => {
+    console.error(`${wasmPath}:`, error)
```

Comment:
@SaltyCucumber if this is not needed, let's revert it

Reply:
Why? There is an import above with the same name, so I've changed this for clarity
the initial issue was with "no such file or directory (os error 2)" and I thought it was a problem with incorrect path, so during debugging I've changed it just in case. Actual problem was with missing wabt lib